lms/djangoapps/commerce/api/v0/urls.py

```diff
@@ -1,9 +1,11 @@
 """
 API v0 URLs.
 """
+from __future__ import absolute_import
```

Comment:
it looks like you have this wherever there are relative imports of commerce modules from within commerce.  Don't explicit relative imports work regardless?

Reply:
Indeed - they do! I've removed them all.